test/e2e/config/k3s-docker.yaml

```diff
@@ -0,0 +1,96 @@
+---
+# E2E test scenario using local dev images and manifests built from the source tree for following providers:
+# - cluster-api
+# - bootstrap k3s
+# - control-plane k3s
+# - docker
+managementClusterName: k3s-e2e
+
+images:
+  # Use local dev images built source tree;
+  - name: ghcr.io/k3s-io/cluster-api-k3s/controlplane-controller:dev
+    loadBehavior: mustLoad
+  - name: ghcr.io/k3s-io/cluster-api-k3s/bootstrap-controller:dev
+    loadBehavior: mustLoad
+
+providers:
+  - name: cluster-api
+    type: CoreProvider
+    versions:
+      - name: v1.6.2
+        value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.6.2/core-components.yaml
+        type: url
+        files:
+          - sourcePath: "../data/shared/v1beta1/metadata.yaml"
```

Comment:
Let's try point to github release file as well.

Reply:
this is not possible, have added more comment about `metadata.yaml`

Reply:
ok, that's fine. Let's update metadata and k3s-docker file regularly to catch up CAPI release.